### PR TITLE
#159951818 Add Password reset feature

### PR DESCRIPTION
**Pull request by abulojoshua1**
### What does this PR do?
- Adds the password reset feature to Author's Haven Frontend

### Description of Task to be completed?
- This feature makes a user able to reset their passwords using a password reset link sent to their emails

### How should this be manually tested?
-  Clone the app.
- Navigate to the login feature and click reset your password
- Type in a registered user's email and submit
- click the reset link sent and fill in the forms visible and submit
- You should be presented with a login modal, then you can log in again

### Any background context you want to provide?
- I created a password reset component
- Added it to the login feature and to the Homepage

### What are the relevant pivotal tracker stories?
- [#159951818](https://www.pivotaltracker.com/story/show/159951818)

### CODE REVIEW

**Comment by geofrocker** [comment]
Rebase this PR, You should be having a different image

File: src/CSS/Loader.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  @keyframes lds-facebook {
+    0% {
+      top: 6px;
+      height: 51px;
+    }
+    50%, 100% {
+      top: 19px;
+      height: 26px;
+    }
+  }
```
Comment (geofrocker) [inline src/CSS/Loader.css]: missing EOF

Comment (abulojoshua1) [reply src/CSS/Loader.css]: Let me fix that now

File: src/tests/containers/Home.test.js [hunk 2]
```diff
@@ -1,14 +1,37 @@
...
 import Popular from '../../components/Popular/Popular';
 import Recent from '../../components/Recent/Recent';
 import Footer from '../../components/Footer/Footer';
 
+it('test check if url token lenght is less', () => {
+  const wrapper = shallow(<Home showLandingPage={true} toggleLandingPageHandler={() => {}} />);
+  expect(wrapper.find(Header)).toHaveLength(1);
+});
+
+// it('test password reset after mail link is clicked', () => {
```
Comment (geofrocker) [inline src/tests/containers/Home.test.js]: remove commented code

**Comment by abulojoshua1** [comment]
I will rebase asap

File: src/tests/containers/Home.test.js (hunk 2 above)
Comment (abulojoshua1) [reply src/tests/containers/Home.test.js]: I will take that out ASAP

**Comment by geofrocker** [comment]
The password reset link does not redirect to the password change modal

**Comment by abulojoshua1** [comment]
@geofrocker, I think you may have made a missed something, It redirects there.

**Pull request merged**